### PR TITLE
Update deployment README

### PR DESCRIPTION
**Pull request by jimmythompson**
Updates to the README, mainly stripping out talk of GCloud and keeping it generic to any Kubernetes installation.

Instructions for how we interact with DigitalOcean are in srobo/infrastructure.

### CODE REVIEW

File: _env/README.md [hunk 1]
```diff
@@ -1,67 +1,28 @@
...
 ## Deploy the application
 
-Todo: Template this somehow
-
-Within `kubernetes/service.yml`, you need to replace the static IP we use
-(i.e. `104.155.107.130`) with the one you intend to use to expose your
-application. Then you run:
-
 ```bash
 kubectl create -f kubernetes/
```
Comment (PeterJCLaw) [inline _env/README.md]: Is this line still useful? (I'm assuming this is part of the initial deployment -- it's not something I've ever run to deploy updates)

Comment (jimmythompson) [reply _env/README.md]: It depends if you've changed any of the config in that directory.

**Comment by jimmythompson** [comment]
@PeterJCLaw I've added details on how to log into DigitalOcean, if you don't mind taking a look. 👍

File: _env/README.md (hunk 1 above)
Comment (PeterJCLaw) [reply _env/README.md]: Ah, that might be worth saying explicitly?

File: _env/README.md [hunk 2]
```diff
@@ -1,67 +1,63 @@
...
+# Deploying the Website
 
 ## Setting up
 
-To setup, you need to have the following installed:
+This assumes you've got access to a kubernetes cluster. We have [a guide][guide]
+on how we do this at Student Robotics.
 
-* [GCloud][gcloud]
+### Deploy the application for the first time
```
Comment (PeterJCLaw) [inline _env/README.md]: Not sure this being h3 makes sense (given the previous heading is h2 and this isn't really a sub-section)?

Comment (jimmythompson) [reply _env/README.md]: I fixed this up in a new commit.

File: _env/README.md (hunk 1 above)
Comment (jimmythompson) [reply _env/README.md]: Turns out the command is subtly different when updating an existing deployment in Kubernetes, I've added this to a new commit.

File: _env/README.md (hunk 2 above)
Comment (PeterJCLaw) [reply _env/README.md]: https://github.com/srobo/website/pull/155/commits/b4da306d387d209967dbc3f77407b0f4d3b13d23

File: _env/README.md (hunk 1 above)
Comment (PeterJCLaw) [reply _env/README.md]: https://github.com/srobo/website/pull/155/commits/b4da306d387d209967dbc3f77407b0f4d3b13d23

**Pull request merged**